superduperdb/container/model.py

```diff
@@ -479,7 +501,10 @@ def on_create(self, db: DB):
         db.create_output_table(self)
 
     def _validate(
-        self, db: DB, validation_set: t.Union[Dataset, str], metrics: t.Sequence[Metric]
+        self,
```

Comment:
and this neither?

Reply:
I made sure to run `make fix-and-test`. So not sure what I can do.

Reply:
may be you have a global setting thing